data/events/2022-washington-dc.yml

```
@@ -236,6 +238,8 @@ sponsors:
     level: community
   - id: kubeevents
     level: community
+  - id: aws-community-day-dmv
```

```suggestion
  - id: awscommunityday-dmv
```

To match the data and image name